src/Backups/BackupsWorker.cpp

```diff
@@ -221,7 +221,7 @@ class BackupsWorker::ThreadPools
             {
                 metric_threads = CurrentMetrics::BackupsThreads;
                 metric_active_threads = CurrentMetrics::BackupsThreadsActive;
-                metric_active_threads = CurrentMetrics::BackupsThreadsScheduled;
```

Comment:
There was a mistake causing metric `BackupsThreadsScheduled` to be used instead of `BackupsThreadsActive`.